api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -106,27 +100,16 @@ public void setUserProvider(Provider<User> userProvider) {
 
     org.pmiops.workbench.notebooks.model.Cluster fcCluster;
     try {
-      fcCluster = this.notebooksService.getCluster(project, DEFAULT_CLUSTER_NAME);
+      fcCluster = this.notebooksService.getCluster(project, NotebooksServiceImpl.getDefaultClusterName());
```

Comment:
Please avoid reaching into an Impl class directly. Move this onto the interface? Constant somewhere?

Reply:
I can try to move this into the interface. Or maybe make it a config variable? I was a little wary of adding too much to the config.